pam_authz/pam/README.md

```diff
@@ -0,0 +1,64 @@
+# Styra's PAM authorization module
+
+This implements a PAM module that integrates with Open Policy Agent (OPA)
+to make authorization decisions. When invoked, the module queries OPA
+and if no errors are returned, then the action is allowed. This allows
+administrators to implement fine-grained access control. For
+example, one can use this module to limit usage of `sudo` to particular
+times of day.
+
+This module is not intended to perform authentication. One can use standard
+linux accounts for that or a user database such as LDAP.
+
+
+## Usage:
```

Comment:
Might be useful to split this section into two parts:

1. How do I build PAM module?
2. How do I install and configure the PAM module?

For (1), we should list prerequisites.